### ISSUE TITLE
Create snippets webview

### ISSUE THREAD

**Issue by RandomFractals** [RandomFractals/vscode-snippets-viewer#5]
- Provide snippet language and loaded snippet extensions filters
- Display snippet prefix, name, description and body in markdown table

**Comment by usernamehw** [comment]
Is it a webview editor or webview view?

**Comment by RandomFractals** [comment]
It will be a webview to view snippets list by language or extension in tabular format.

**Comment by usernamehw** [comment]
I don't like how `body` looks in that extension. Will `body` column have formatting and syntax highlighting?

**Comment by RandomFractals** [comment]
yep. I'll probably use this lib to make it pretty: https://highlightjs.org/

**Comment by usernamehw** [comment]
https://github.com/shikijs/shiki - This highlighter is based on `vscode-textmate`, so the highlighting is almost like native vscode one , but I'm not sure it can work in a webview (Although it kind of should: https://marketplace.visualstudio.com/items?itemName=bierner.markdown-shiki)

**Comment by usernamehw** [comment]
Actually, it probably wouldn't work. That extension uses shiki on extension side and provides syntax highlighting for markdown preview.

**Comment by RandomFractals** [comment]
I'll take a look. Thanks for the links. Ideally, I would like to have snippet body formatted using MarkdownString vscode uses, but I don't know how to wire that in webview yet :)
